frontend/packages/console-app/src/utils/access-mode.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { RadioInput } from '@console/internal/components/radio';
+import { FormGroup } from '@patternfly/react-core';
+import {
+  provisionerAccessModeMapping,
+  getAccessModeForProvisioner,
+  cephRBDProvisionerSuffix,
+  accessModeRadios,
+} from '@console/internal/components/storage/shared';
+import { PersistentVolumeClaimKind } from '@console/internal/module/k8s';
+
+export const AccessModeSelector: React.FC<AccessModeSelectorProps> = (props) => {
```

Comment:
better place would be `console-app/src/components/access-mode` since its a component